Veverka_statistics ui

alltimeJoe216 opened the pull request:
## Testing
No tests!
## Style Guide
Following Lisa's wireframe
## Feature
Added and refactored necessary views in Production reporting
## Dependencies
N/A
Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings

alltimeJoe216 committed:
Constraints for VCs

alltimeJoe216 committed:
Added VC file for existing reports. Constraints look good in Xcode, but I can't get to the view on simulator due to not being able to login.

supergonzo812 commented on labs-ios-starter/View Controllers/Impact Statistics/Production Reports/ExistingReportViewController.swift:
```diff
@@ -0,0 +1,48 @@
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
```
Delete unnecessary white space.

supergonzo812 commented on labs-ios-starter/View Controllers/Impact Statistics/Production Reports/ExistingReportViewController.swift:
```diff
@@ -0,0 +1,48 @@
+    @IBOutlet var reportImages: UIImageView!
+    @IBOutlet var deleteButtonOutlet: UIButton!
+    @IBOutlet var editButtonOutlet: UIButton!
+    
```
Create another Mark for View Life Cycle, so it does not seem that the ViewDidLoad is part of the IBOutlets.

alltimeJoe216 committed:
Cleaned up code in accordance with Chris' comments. Added image picker to production report. Not tested yet...
